src/DetourContext.tsx

```diff
@@ -21,7 +21,9 @@ export const useDetourContext = () => {
   const context = useContext(DetourContext);
 
   if (!context) {
-    throw new Error('useDetourContext must be used within a DetourProvider');
+    throw new Error(
+      'Detour: useDetourContext must be used within a DetourProvider'
```

Comment:
I think we should print it in brackets, for example something like that 

throw new Error("🔗[Detour:NETWORK_ERROR] Failed to fetch deferred link (timeout).");